src/donation.html

```diff
@@ -0,0 +1,116 @@
+---
+layout: nofooter.html
+pageTitle: Donation
+tags: nav
+order: 5
+---
+<h1 class="page-title">Donation</h1>
+
+<ul class="donation-sub">
+    <li>Drop-off donations time:</li>
+    <li>9Am-5Pm Thursday - Sunday</li>
+    <li>Pick-ups time:</li>
+    <li>7Am-2Pm Monday - Friday</li>
+</ul>
+
+<ul class="donation-sub">
+    <li>Call us: 410-385-1700</li>
+    <li>Email: donations@secondchanceinc.org</li>
+</ul>
+
+<h2>Donation Form</h2>
+<section class="form">
+    <label for="name">Name</label>
+    <input id="name" type="text" placeholder="Required">
```

Comment:
Required form elements can use the HTML attribute `required`, like so:

```suggestion
    <input id="name" type="text" placeholder="Required" required>
```